recipes/pyhough/meta.yaml

```diff
@@ -1,47 +1,46 @@
 {% set name = "pyhough" %}
 {% set version = "0.0.7" %}
+{% set python_min = "3.8" %}
```

Comment:
Only set `python_min` if the minimum required version is higher than the conda-forge default. 

```suggestion
```